### PR TITLE
set up project 1 into.

### PR DESCRIPTION
**Pull request by aliabbani**
## Hello Mcroverse Project: set up project intro.

In this milestone we:
- added a new branch
- added linters
- added HTML & CSS files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Hello Microverse first project.
+
+![screenshot](./app_screenshot.png)
```
Comment (vichuge) [inline README.md:7]: - [ ] This image is broken, on this case, I think the image is not necessary, you could delete this line. 🐶

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (vichuge) [inline .gitignore:1]: - [ ] because node_modules is a folder,  you could use `node_modules/`. 🚗

Comment (vichuge) [inline .gitignore:1]: - [ ] Please, add `test.md` on the .gitignore file. 🐦

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [ ] If you add a space after the script, you could see the bullets on the description.
- For
- example
- this
- just use `- For` `- example` `- this` (note the space between the script and word.
image

- [ ] The pull request (PR) title could be more descriptive, `Feature-1 branch` doesn't say what you do on this PR. I kindly invite you to be more descriptive.

- [ ] The branch name `feature-1` could be more descriptive too.

- [ ] Check the comments under the review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 2 above)
Comment (aliabbani) [reply .gitignore:1]: how?

**Pull request closed without merging**